docs/organisation-logos.md

```diff
@@ -0,0 +1,43 @@
+# Organisation logos
+
+## Overview
+
+The [Organisation logo component](https://components.publishing.service.gov.uk/component-guide/organisation_logo) can be used to render organisation logos that use either the [Government Identity System](https://gcs.civilservice.gov.uk/guidance/marketing/branding-guidelines/), such as [HMRC](https://components.publishing.service.gov.uk/component-guide/organisation_logo/hm_revenue_customs/preview), or have a [brand exemption](https://gcs.civilservice.gov.uk/guidance/marketing/branding-guidelines/#Brand-exemption-process), such as [HMPS](https://components.publishing.service.gov.uk/component-guide/organisation_logo/hm_prison_service/preview).
+
+This documentation provides details on how to update and add new logos that are part of the Government Identity System.
+
+## Images required
+
+The four images below are required for the organisation logo component. The image format required is a transparent PNG with the whitespace trimmmed.
+
+### Mobile
+
+By default, the logo image used on mobile screen sizes, will have a height of `20px`.
+
+- `#{$crest}_13px.png`
+- `#{$crest}_13px_x2.png`
```

Comment:
I was about to ask about the conflicting size/naming convention when I read ahead to the explanation. Maybe move that explanation to the start, so it's more prominent?

Reply:
Thanks, that makes sense, I've moved it to the start